tox.ini

```diff
@@ -69,3 +70,20 @@ deps =
   pytest-mock
 commands = 
   pytest -v --tb native {[vars]scripts_test_path}/test_branch_track_creation.py --log-cli-level=INFO -s {posargs}
+
+[testenv:test_1dot4]
+description = Test bundles
+deps = 
+  lightkube
+  pytest
+  pytest-operator
+commands = pytest -vs --tb native {[vars]releases_test_path} -m "v1dot4" --model kubeflow {posargs}
+
+
+[testenv:test_1dot6]
+description = Test bundles
+deps = 
+  lightkube
+  pytest
+  pytest-operator
+commands = pytest -vs --tb native {[vars]releases_test_path} -m "v1dot6" --model kubeflow {posargs}
```

Comment:
There might be a really succinct way to make a single generic `testenv` that covers all versions and `full` vs `lite`, etc.   I don't know tox well enough to guess at the syntax though.

`commands` in tox envs can use the env name, or even parts of the env name.  Check out the [kfp-operators tox.ini](https://github.com/canonical/kfp-operators/blob/main/tox.ini) for some ideas.  It specifies an envlist and extracts data from the env name, although how it works gets a bit odd imo.  Your case is a bit different, but maybe there's still a way.  Naming envs and pytest marks the same might make this easier at least.